docroot/themes/humsci/su_humsci_theme/scss/nodes/hs_person.scss

```diff
@@ -5,55 +5,41 @@
 //
 
 // Decanter
-@import 'decanter-no-markup';
+@import
+  'decanter-no-markup';
 
-.email-office-wrapper {
-
-  > div {
-    float: left;
-
-    &:last-child {
-      float: none;
-    }
+.hs-person {
+  .field-block {
+    @include margin(null null 0.5rem 0);
   }
 
-  .divider {
-    margin: 0 5px;
-  }
-}
-
-.hs-person.full.node {
+  .field-hs-person-title h3 {
```

Comment:
I think this should actually be more universal on the layout. lets add this to https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/modules/humsci/hs_layouts/scss/three-column-w-image.scss and target all headings in the two regions that wrap the image so we don't have to override them on each node